website/docs/r/compute_vm_instance.html.markdown

```diff
@@ -33,6 +33,7 @@ resource "ibm_compute_vm_instance" "twc_terraform_sample" {
     local_disk = false
     public_vlan_id = 1391277
     private_vlan_id = 7721931
+    public_security_groups = ["allow_ssh"]

```

Comment:
it looks like you have `public_security_groups` here with names, but it looks like you have `public_security_group_ids` below with int ids.